### DISCUSSION TITLE
Configuring the jwt-secret for postgrest

### DISCUSSION THREAD

**Discussion by lmcardle** [supabase/supabase#3559] [Questions]
Asking this here as I’m not quite sure the best place to ask this.

Is there a way to provide the ‘ jwt-secret’ which Postgrest should use for JWTs? Specifically I’m trying to integrate Supabase with an existing auth system (I happen to use Auth0) and thus want to enable RLS leveraging this existing user JWT. 

It would also be great if it is possible to set the audience claim (https://postgrest.org/en/v8.0/configuration.html#jwt-aud), but I'd certainly settle for just the jwt-secret (https://postgrest.org/en/v8.0/configuration.html#jwt-secret) at this time.

Looks like a similar question was asked about a year ago (supabase/supabase-js#25 (comment)) but it doesn’t look like there was a resolution. @awalias you made the comment “we should build that in” so hope you don’t mind me tagging you here.

Thanks in advance.

**Comment by awalias** [comment]
hey @lmcardle ! Yes you are correct, we still haven't added the ability to set custom jwt-secrets yet, I will bump it up our list now
